src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+    
+    public boolean isMutating() {
```

Comment:
Header comment for methods to be overriden is necessary for others to follow the pattern. Also think about how to enforce the overriding behaviour in the subclasses. Hint: adding one keyword to the method signature will work.